virtual/openssh/openssh-9.3_p1.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Virtual for net-misc/openssh and variants"
+
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~loong ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="hpn sctp ssl X509"
+
+RDEPEND="
+	|| (
+		~net-misc/openssh-${PV}[hpn(-)?,sctp(-)?,ssl(-)?,X509(-)?]
+		~net-misc/openssh-contrib-${PV}[hpn(-)?,sctp(-)?,ssl(-)?,X509(-)?]
```

Comment:
I think you actually need `=` for these flags. There's one revdep that does `-something` dep on virtual, so this dep wouldn't enforce it on the underlying implementation.